CONTRIBUTING.MD

```diff
@@ -0,0 +1,81 @@
+# Contributing guide
+
+Thanks for your contribution 🥳 to the Fastily website redesign project!
+
+[Code of Conduct](https://github.com/fastify/.github/blob/main/CODE_OF_CONDUCT.md)
+
+* [Vocabulary](#vocabulary)
+* [📝 Commit message guidelines](#-commit-message-guidelines)
+* [📜 Pull Request Policy](#-pull-request-policy)
+* [Before merging](#before-merging)
+* [When merging](#when-merging)
+* [Becoming a collaborator](#becoming-a-collaborator)
+* [🐛 Debugging failing checks](#-debugging-failing-checks)
+* [Developer's Certificate of Origin 1.1](#developers-certificate-of-origin-11)
```

Comment:
Could we add emojy everywhere or nowhere?
It seems confusing this random thing

Reply:
I tried to put some in to make it less boring.